test/banners/desktop/WMDE_FR_2025_Desktop_DE_09/components/BannerCtrl.spec.ts

```diff
@@ -135,15 +134,6 @@ describe( 'BannerCtrl.vue', () => {
 		} );
 	} );
 
-	describe( 'Soft Close Submit Tracking', () => {
-		test.each( [
-			// this ctrl banner does not have the softclose feature
-			[ 'expectStoresCloseChoiceInBannerWithoutSoftClose' ]
```

Comment:
Instead of removing this test completely, you should include the `expectEmitsBannerSubmitOnReturnEvent` and `expectDoesNotEmitsBannerSubmitOnReturnEventWhenLocalStorageItemIsMissing` in the tests for both banners. The description, "Soft Close Submit Tracking" is still correct, because we're tracking the submit behavior when of the stored soft close interactions of *previous* banners.